README.md

```diff
@@ -72,15 +72,16 @@ It also has several useful helpers to handle errors.
 
 | Param | Type | Default | Description |
 | --- | --- | --- | --- |
-| auth | `Object <username, password, targetBranch>` | - | See `auth` description [below](#auth). |
+| auth | `Object <username, password, targetBranch, pcc, overridePcc>` | - | See `auth` description [below](#auth). |
```

Comment:
I see two new params added in docs: `pcc` and `overridePcc`.
I can see what is `overridePcc` responsible for, but what about `pcc`?
I found it's only used in Air service and only in this line:
https://github.com/Travelport-Ukraine/uapi-json/blob/master/src/Services/Air/AirService.js#L25
(and this variable is used then here: https://github.com/Travelport-Ukraine/uapi-json/blob/master/src/Services/Air/templates/AIR_CREATE_RESERVATION_REQUEST.xml#L53)

Do you think, guys that `pcc` should really be passed in `auth`?
I can see clear reasons why `overridePcc ` should but not `pcc`.
@shmuga , @jirismika ?